.env.example

```diff
@@ -1,16 +1,24 @@
-NODE_ENV=development
+
 PORT=3000
 HOST=http://localhost:3000
 API_URL=https://api.test.faforever.com
 OAUTH_URL=https://hydra.test.faforever.com
-WP_URL=https:direct.test.faforever.com
-
+WP_URL=https://direct.faforever.com
+CALLBACK=auth
 OAUTH_CLIENT_ID=faf-website
 OAUTH_CLIENT_SECRET=banana
 SESSION_SECRET_KEY=banana
 RECAPTCHA_SITE_KEY=6Lc4GmgaAAAAAPaFbnwuP_dBGG--SSy2Hi5UcqwK
 
+
 TOKEN_LIFESPAN=43200                       # an api token is valid for 12h
+LEADERBOARDS_UPDATE_INTERVAL=900          # 15 minutes
+RECENT_USERS_LIST_UPDATE_INTERVAL=900     # 15 minutes
+CLIENT_RELEASE_FETCHING_INTERVAL=900      # 15 minutes
+
```

Comment:
Are these supposed to be back?

Reply:
No, I'm not sure why they propped back, probably an error of mine while resolving the conflicts...